### ISSUE TITLE
html in php files

### ISSUE THREAD

**Issue by robo-wg** [prettier/prettier-vscode#1055]
would be great if prettier would also work in php files. I mean at least regarding html in php files.
thanks for this great extension

**Comment by kachkaev** [comment]
Formatting should work if you install Prettier and its [PHP extension](https://github.com/prettier/plugin-php) as dependencies of your project. It'd be great if VSCode extension also supported discovery of global Prettier and its plugins. I'm not programming in PHP these days, but I have a [similar need](https://github.com/prettier/prettier-vscode/issues/958#issuecomment-531334953) for Elm projects. Having local `package.json` `yarn.lock` and `node_modules` is the best option for every situation.

**Comment by ntotten** [comment]
@kachkaev is correct. Prettier will work if you install the PHP plugin. This extension will pick it up when you install the extension.

**Comment by robo-wg** [comment]
hello and thanks for quick replying.
But the command 

`npm install prettier @prettier/plugin-php`

unfortunately doesn't work
I'm getting the error(s) right now:

npm ERR! code ENOGIT
npm ERR! Error while executing:
npm ERR! undefined ls-remote -h -t ssh://git@github.com/glayzzle/php-parser.git
npm ERR! 
npm ERR! undefined
npm ERR! No git binary found in $PATH
npm ERR!
npm ERR! Failed using git.
npm ERR! Please check if you have git installed and in your PATH.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mengh\AppData\Roaming\npm-cache\_logs\2019-11-14T23_35_37_682Z-debug.log

**Comment by kachkaev** [comment]
Seems like you don't have git installed: `No git binary found in $PATH`.

1. Install [`git`](https://git-scm.com/).
1. Make sure you can run `git --help` in your terminal and it does not produce an error.
1. Try `npm install prettier @prettier/plugin-php` again.

This is not a bug in Prettier VSCode extension. If you struggle getting through the above steps, feel free to open an issue in https://github.com/prettier/plugin-php

**Comment by robo-wg** [comment]
Hi kachkaev,
you were right. git was missing. Thanks for this note!
I installed the PHP extension now, but it still doesn't work -> format selection is not displayed
I installed it as dev dependencies from my project and I tried the global installation as well.

[i will continue here](https://github.com/prettier/plugin-php/issues/1213) ... this is exactly the same problem. 
Thank you very much,
Robert
